### ISSUE TITLE
Title above the main menu not working

### ISSUE THREAD

**Issue by Chudz** [BeSite/jQuery.mmenu#359]
Used the Playground to create the code and basically copied and pasted it all.

The main manu title I set which was Main Menu does not show up in the menu on my site but it does on the playground.
```
	   "extensions": [
	      "border-full",
	      "effect-slide-menu",
	      "effect-slide-panels-100",
	      "pageshadow",
	      "theme-dark"
	   ],
	   "offCanvas": {
	      "position": "right"
	   },
	   "navbars": [
	      {
	         "position": "top",
	         "title": "Main Menu"
	      }
	   ]
	});
```

**Comment by FrDH** [comment]
That seems correct, are you sure the navbars JS and CSS files are included on the webpage?

**Comment by Chudz** [comment]
Yeah, you can see it here: http://www.cqwebdesign.co.uk/paul/
jquery.mmenu.all.css is included in the style.css file. I merge them using Sass

**Comment by FrDH** [comment]
You're using a (slightly) outdated version of the plugin :-)
The "navbars" add-on is available for version 5.1 and up, older versions have the "header" add-on:
````javascript
$("#menu").mmenu({
   header: {
      add: true,
      update: true,
      title: "Main Menu"
   }
});
````

**Comment by kriskd** [comment]
I am having a similar issue.  I am using version 5.2.0 of the plugin.  The HTML for the title is in my DOM, but the ul with the menu is covering it.  If I manually put in a margin-top: 19px with the inspector, it drops down appropriately and the title appears and work to go to the previous level of the menu.
